bin/lambda/build.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
```

Comment:
The `.sh` extension is not required thanks to this shebang. `bin/lambda/build` would be lovely.